app/views/facilities/show_patients.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="bg-white">
+  <div class="px-4 py-8 max-w-7xl mx-auto sm:px-6 lg:px-8">
+    <div class="pb-5 border-b border-gray-200">
+      <h3 class="text-lg leading-6 font-medium text-gray-900">
+        <%= "Patients under #{@facility.name} facility" %>
+      </h3>
+      <p class="mt-2 max-w-4xl text-sm text-gray-500">Click on More Info to see more details about the patient.</p>
+    </div>
+  </div>
+</div>
+<section aria-labelledby="notes-title">
+  <div class="bg-white shadow sm:rounded-lg sm:overflow-hidden">
+    <div class="divide-y divide-gray-200">
+      <div class="px-4 py-5 sm:px-6">
+        <h2 id="notes-title" class="text-lg font-medium text-gray-900">Patients</h2>
+      </div>
+      <div class="px-4 py-6 sm:px-6">
+        <ul class="space-y-8 list-disc ml-4">
+          <% @facility.patients.each do |patient| %>
+            <li class="pl-3 pr-4 py-3 flex items-center justify-between text-sm">
+              <div class="w-0 flex-1 flex items-center">
+                <svg class="flex-shrink-0 h-5 w-5 text-gray-400" x-description="Heroicon name: solid/paper-clip" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
+                  <path fill-rule="evenodd" d="M8 4a3 3 0 00-3 3v4a5 5 0 0010 0V7a1 1 0 112 0v4a7 7 0 11-14 0V7a5 5 0 0110 0v4a3 3 0 11-6 0V7a1 1 0 012 0v4a1 1 0 102 0V7a3 3 0 00-3-3z" clip-rule="evenodd"></path>
+                </svg>
```

Comment:
From this #29, we have moved to FA for Icons. So the SVGs could be replaced. Kindly make a note of it for other files too. 📍

Reply:
✅ Done as of [this commit](https://github.com/coronasafe/arike/pull/48/commits/8e59559238006ff887eca962475168b1e60a6c50#diff-3262c0612e7add3d75cce[…]87bbe7530e163d9e21456cbR22)